packages/auth/utils/recovery.ts

```diff
@@ -1,17 +1,18 @@
 import type { Account } from '@walless/graphql';
 import { mutations } from '@walless/graphql';
+import { modules } from '@walless/ioc';
 import BN from 'bn.js';
 
-import { injectedModules } from './ioc';
+import { key } from './w3a';
 
 export const initAndRegisterWallet = async (): Promise<Account | undefined> => {
 	try {
-		await injectedModules.key.reconstructKey();
-		const newShare = await injectedModules.key.generateNewShare();
+		await key().reconstructKey();
+		const newShare = await key().generateNewShare();
```

Comment:
why do we need key as a method?

Reply:
its a way to bind aliased type for `never`.